sky/skypilot_config.py

```diff
@@ -673,6 +674,19 @@ def apply_cli_config(cli_config: Optional[List[str]]) -> Dict[str, Any]:
     return parsed_config
 
 
+def force_override_config_file(override_configs: Dict[str, Any]) -> None:
+    """Force overrides the config.
+
+    This is only for internal use.
+    """
+    original_config = _get_loaded_config()
+    for k, v in override_configs.items():
+        original_config[k] = v
+    with open(os.path.expanduser(_GLOBAL_CONFIG_PATH), 'w',
+              encoding='utf-8') as f:
+        yaml.safe_dump(dict(original_config), f)
```

Comment:
This might not work with a helm deployment since the source of skypilot config is managed as a kubernetes configmap